client/src/components/atoms/BannerCarousel.tsx

```diff
@@ -0,0 +1,27 @@
+import { Carousel, Container } from "react-bootstrap";
+import React, { FunctionComponent } from "react";
+import DonorHomepageConfig from "../../config/donorHompageConfig.json";
+
+const BannerCarousel: FunctionComponent = () => {
+    const bannerImgs = DonorHomepageConfig.Banner;
```

Comment:
Read the `DonorHomepageConfig` in a wrapper component for `BannerCarousel`
Then pass data (image urls, banner text) to `BannerCarousel`

We want to make `BannerCarousel` reusable